### PR TITLE
Implement DI container

### PR DESCRIPTION
**Pull request by tarnawski**

### CODE REVIEW

**Comment by tarnawski** [comment]
@partikus @dominikhajduk @PorkOne please CR if you have time

File: app/Application.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        }
+    }
+
+    public function loadRoutes(string $path)
+    {
+        $data = yaml_parse_file($path);
+        $this->routes = $data['routes'];
+    }
+
+    public function loadServices()
```
Comment (dominikhajduk) [inline app/Application.php]: @tarnawski Shouldn't we grab services from config files? Currently when you want add service you need edit `loadServices`.

Other changes looks fine for me.

Comment (tarnawski) [inline app/Application.php]: Done

File: app/ContainerInterface.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * @param array $parameters
+     */
+    public function registerParameters(array $parameters);
+
+    /**
+     * @param array $services
+     */
+    public function registerServices(array $services);
+}
```
Comment (dominikhajduk) [inline app/ContainerInterface.php]: @tarnawski please run cs fixer for missing end lines

**Pull request merged**